CloudOnMobile/Modules/Files/Models/File.swift

```diff
@@ -7,13 +7,31 @@
 
 import Foundation
 
-struct File {
+struct File: Codable {
```

Comment:
We can create a separate model `BackendFileModel` or sth to be Codable.
File will be representation for our needs for views and app not backend.